source/wp-content/themes/wporg-parent-2021/patterns/two-columns-with-cta.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Title: 2 Columns with CTA
+ * Slug: wporg-parent-2021/two-columns-with-cta
+ * Categories: columns, buttons
+ */
+
+?>
+<!-- wp:columns {"align":"full","style":{"border":{"top":{"color":"var:preset|color|light-grey"}}},"className":"is-style-default"} -->
+<div class="wp-block-columns alignfull is-style-default" style="border-top-color:var(--wp--preset--color--light-grey)"><!-- wp:column {"width":"100%","style":{"spacing":{"padding":{"top":"80px","right":"100px","bottom":"100px","left":"100px"}}}} -->
+<div class="wp-block-column" style="padding-top:80px;padding-right:100px;padding-bottom:100px;padding-left:100px;flex-basis:100%"><!-- wp:heading {"level":4} -->
```

Comment:
Padding needs responsive overrides on mobile

Reply:
Is there a way for us to use `REM` units for all the copy/heading and then utilize `em` units for padding? Extend the column block to allow us to set font-sizes for different breakpoints to the parent container.

Reply:
You can use the padding preset properties, which we can update to use clamp or have responsive changes in the CSS. [See `parts/cover.html` for an example](https://github.com/WordPress/wporg-parent-2021/blob/7acf81d3297b4327177cb7e5df885269c538280a/source/wp-content/themes/wporg-parent-2021/parts/cover.html#L1-L2), but the attribute uses his format: `"padding":{"top":"var:preset|spacing|60",…`, and then the inline CSS uses the property, `padding-top:var(--wp--preset--spacing--60)`.

I listed out some of the defined properties here: https://github.com/WordPress/wporg-parent-2021/pull/18#issuecomment-1199977183

Reply:
Thanks for this. I've nearly got it working but for some reason when I insert the pattern the columns are full width rather than wide, even though the class is in the markup:

```
<!-- wp:columns {"align":"wide","style":{"spacing":{"blockGap":"0px"}},"className":"two-columns-cta"} -->
<div class="wp-block-columns alignwide two-columns-cta">
```

I can't for the life of me work out how to set this... as you can see there is no alignment setting on the block options in the editor. How did you manage to set up this layout?

![Screen Shot 2022-08-02 at 4 58 34 PM](https://user-images.githubusercontent.com/1017872/182295294-4bed24d6-8bba-40a6-8579-01f4dcb9b6c8.jpg)

Reply:
Check the group block that's wrapping the columns - is the Layout toggle on? If so, it overrides the alignment of child elements & forces them to be the full width of the group. If you untoggle that, the alignment options on the Columns should come back.

<img width="281" alt="Screen Shot 2022-08-02 at 10 16 03 AM" src="https://user-images.githubusercontent.com/541093/182396655-eeafbee4-1f1d-4701-af26-da5c88e96ae6.png">

Reply:
Ah that was it, thanks!

Reply:
Updated to use the Large padding with clamping, works quite well